src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14257,5 +14257,76 @@ public void DefaultsParamsConversion_Mix()
                 // D d1 = (int a, int b = 2, params int[] c) => { }; // 1, 2
                 Diagnostic(ErrorCode.ERR_DefaultValueBeforeRequiredValue, ")").WithLocation(1, 41));
         }
+
+        [Fact]
+        public void Lambda_DefaultParameters_BreakingChange()
+        {
+            var source = """
+                void MethodWithDefault(int i = 0) { }
+                var action = MethodWithDefault;
+                DoAction(action, 1);
+                void DoAction(System.Action<int> a, int p) => a(p);
+                """;
+
+            CreateCompilation(source, parseOptions: TestOptions.Regular11).VerifyDiagnostics();
+
+            CreateCompilation(source, parseOptions: TestOptions.RegularNext).VerifyDiagnostics(
+                // (3,10): error CS1503: Argument 1: cannot convert from '<anonymous delegate>' to 'System.Action<int>'
+                // DoAction(action, 1);
+                Diagnostic(ErrorCode.ERR_BadArgType, "action").WithArguments("1", "<anonymous delegate>", "System.Action<int>").WithLocation(3, 10));
```

Comment:
As part of this PR, it would be good to document the breaking change in question.
That should go into [https://github.com/dotnet/roslyn/blob/features/lambda-default-parameters/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%207.md](https://github.com/dotnet/roslyn/blob/features/lambda-default-parameters/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%207.md)

Reply:
Thanks, added: 45fe78fa8d7deb9caf7054539c1e187b77494a73